backend/coins/eth/account.go

```diff
@@ -447,6 +453,7 @@ func (account *Account) Close() {
 		account.log.Info("Closed DB")
 	}
 	close(account.quitChan)
```

Comment:
this should be be guarded too by the lock.

Reply:
oh right